Source/WebCore/platform/mediastream/cocoa/AudioMediaStreamTrackRendererInternalUnit.cpp

```diff
@@ -266,13 +271,17 @@ static void clipAudioBufferList(AudioBufferList& list, AudioStreamDescription::P
 
 OSStatus LocalAudioMediaStreamTrackRendererInternalUnit::render(AudioUnitRenderActionFlags* actionFlags, const AudioTimeStamp* timeStamp, UInt32 sampleCount, AudioBufferList* ioData)
 {
+    RefPtr client = m_client.get();
+    if (!client)
+        return kAudio_ParamError;
```

Comment:
I'm not sure if this is the correct error to return here.

Reply:
I think I would use `kStateErr` (Not in appropriate state to perform operation)

Reply:
Hmm, I tried that and am getting a build error. Where is `kStateErr` defined? Is there a header I need to include?

Reply:
Does it fail locally or only on the bots?

Reply:
It doesn't really matter, just go back to `kAudio_ParamError` if that worked.

Reply:
It was failing both locally and on the bots. I've gone back to using `kAudio_ParamError`.